cfme/services/myservice/ssui.py

```diff
@@ -225,6 +241,20 @@ def launch_vm_console(self, catalog_item):
     return vm_obj
 
 
+@MyService.retire.external_implementation_for(ViaSSUI)
+def retire(self):
+    view = navigate_to(self, 'Retire')
+    view.retire.click()
+    view = self.create_view(MyServicesView)
+    wait_for(
+        lambda: view.is_displayed, delay=15, num_sec=300,
+        message="waiting for view to be displayed"
+    )
+    assert view.is_displayed
+    # TODO - remove sleep when BZ 1496233 is fixed
```

Comment:
Remove this sleep, the BZ is verified.

Reply:
This bug is fixed for 5.9 not for 5.8 , BZ - 1518954

Reply:
Please add both bz numbers to comment